doc/setup/build_install_example.adoc

```diff
@@ -18,8 +19,7 @@ Demonstrates a simple C network proxy that can send and receive OSI data via TCP
 
 **Prerequisites**
 
-* You have installed _cmake_ version 3.10.2 or higher.
-* You have installed _protobuf_ version 3.0.0 or higher.
+* You have installed all prerequisites according to the way of linking of your choice, as described in <<Setting up OSI>>.
```

Comment:
```suggestion
* You have installed all prerequisites according to the way of linking of your choice, as described in xref:setup/setting_up_OSI.adoc#sec-setting-up-osi[].
```

As discussed in CCB 2023-03-13, adjustments to OSI repo to make setup section a separate file (@PhRosenberger and @philipwindecker will adjust PR OpenSimulationInterface/open-simulation-interface#715 to create the separate file).

Reply:
My bad: It should be "top" instead of "sec". I will fix this in the suggestion above.

Reply:
Actually, I would suggest staying with the original syntax for readability here. This is also in line with the ASAM Editorial Guide.
The full path is helpful when trying to point to a specific file. However, if the anchor is unique anyway, the following is easier to create and use:

```suggestion
* You have installed all prerequisites according to the way of linking of your choice, as described in <<top-setting-up-osi>>.
```